_episodes/01-introduction.md

```diff
@@ -6,18 +6,22 @@ questions:
 - "What are containers, and why might they be useful to me?"
 objectives:
 - "Show how software depending on other software leads to configuration management problems."
-- "Identify the problems that software installation problems can pose for research."
-- "Give two examples of how containers can solve software configuration problems." 
+- "Identify the problems that software installation can pose for research."
+- "Explain the advantages virtualization has over traditional computing."
+- "Give two examples of how containers can solve software configuration problems."
 keypoints:
 - "Almost all software depends on other software components to function, but these components have independent evolutionary paths."
-- "Projects involving many software components can rapidly run into a combinatoric explosion in the number of software version configurations available, yet only a subset of possible configurations actually works as desired."
-- "Containers collect software components together and can help avoid software dependency problems."
-- "Virtualisation is an old technology that container technology makes more practical."
+- "Small environments that contain only the software that's absolutely needed for a given task are easier to replicate and maintain."
+- "Critical systems that cannot be upgraded, due to cost, difficulty, etc. need to be reproduced on newer systems in a maintainable and self-documented way."
+- "Virtualization was created to allow multiple environments to run on a single computer."
+- "Containerization takes virtualization one step further by making even smaller environments by avoiding all but the minimum software required."
```

Comment:
I agree that this is the usual effect for many use cases, but the current wording suggests the software minimisation is necessarily the primary objective, although this is not the exclusive goal of the underlying technical mechanisms.  Containerisation makes (hardware) virtualisation "cheaper"... how about something like "Containerisation improves upon the virtualisation of whole computers by allowing efficient management of the host computer's memory and storage resources." ... although that may be too technically worded.

Reply:
```suggestion
- "Containerization improves upon the virtualization of whole computers by allowing efficient management of the host computer's memory and storage resources."
```